ext/certstore/loader.c

```diff
@@ -428,7 +427,7 @@ rb_win_certstore_loader_export_pfx(VALUE self, VALUE rb_thumbprint, VALUE rb_pas
                                         pContext);
   if (!pContext) {
     _snprintf_s(errBuf,
-                256,
+                sizeof(errBuf),
```

Comment:
nit: This parameter should be byte size or element count?

byte size: sizeof
element count: [_countof](https://docs.microsoft.com/ja-jp/cpp/c-runtime-library/reference/countof-macro?view=vs-2019)

Reply:
Results of them are same in multi byte string.
`char` (a.k.a `CHAR` in Win32) is 1 byte, it's C lang's spec.